### DISCUSSION TITLE
Add toleration support for Deployment resources

### DISCUSSION THREAD

**Discussion by trillejs** [fluxcd/flux2#895] [Proposals]
Hi,
[I originally raised the issue on the flux Terraform provider repository but I was said to transfer it here](https://github.com/fluxcd/terraform-provider-flux/issues/93):

I am trying to deploy flux and I saw that it is possible to add custom annotations/labels/etc... with a kustomize patch after flux is deployed: https://toolkit.fluxcd.io/guides/installation/#customize-flux-manifests

However the source controller is not deploying because there is no way for me to add tolerations to the node taints I want to deploy it to.

Is there a way of doing this that I am missing?

**Comment by stefanprodan** [comment]
I think tolerations are a good candidate for being part of flux install options and Terraform provider.

**Reply by phillebaba** [reply]
I think this would be a good option, and as long as it is an option in manifest gen it can easily be added to the datasource.

**Comment by stefanprodan** [comment]
Implemented in https://github.com/fluxcd/flux2/pull/899
